chromadb/cli/commands/estimate.py

```diff
@@ -0,0 +1,182 @@
+from typing import Tuple, Union
+import typer
+import humanize
+from rich.console import Console
+from rich.table import Table
+
+console = Console()
+
+mem_cpu_mapping = {1: 1, 2: 1, 4: 2, 8: 4, 16: 8, 32: 16, 64: 32, 128: 64}
+common_vector_dims = {1536: ["OpenAI"], 768: [
+    "BERT"], 512: ["ResNet"], 368: ["all-MiniLM-L6-v2"]}
+
+# percentage of memory used for system memory, python process (e.g. SQL queries etc)
+overhead_factor = 0.3
```

Comment:
Have we empirically verified this a bit / can we be more systematic about usage of sqlite metadata segments?